src/menu.rs

```diff
@@ -175,17 +178,20 @@ pub fn run(
             match event {
                 Input::Render(args) => {
                     opengl.draw(args.viewport(), |context, graphics| {
+                        clear(color::BLACK, graphics);
```

Comment:
Instead of changing the args of the menu draw function , I pulled out the clear call and added the sprite scene draw before the rest of the drawing. Could throw it in that function too of course. Unless we convert EVERYTHING to sprites and just add them all to the scene.

Reply:
Can we just pass the `scene` to `draw()` so that at least all of the drawing is encapsulated nicely in a single function?